### ISSUE TITLE
Solace message does not have an id mapping between JMSToolBox and Solace UI

### ISSUE THREAD

**Issue by solace-user** [jmstoolbox/jmstoolbox#179]
## Specifications
- JMSToolBox version: 5.12.0
- OS (Name, version, 32/64bits): Windows 10 / Windows Server 64 bits
- Q provider (kind, version) : Solace, 10.7.1.51

## Expected Behavior
When we want to use JMSToolBox to browse the message content, we cannot find something unique id linked to the Solace UI. For example, in Solace we can have a message ID to classify that message. However, in JMSToolBox we can only find JMSMessageID but it does not really relate to the message ID shown in Solace UI. For now what we can classify the message is by the datetime and the payload length. But this does not make sense that not to have a unique ID to map with each other.

## Actual Behavior
In JMSToolBox we can only find JMSMessageID but it does not really relate to the message ID shown in Solace UI

## Steps to Reproduce the Problem

  1. Open JMSToolBox and search the queue.
<img width="1447" alt="solace_JMSToolBox" src="https://github.com/user-attachments/assets/876ff89f-c19f-4658-8b4b-fe2d833244b5">

## Logs
Eventually, attach the log files which are `"<user>\.jtb\JMSToolBox\jmstoolbox.log"` and `"<user>\.jtb\.metadata\.log"`    
They can be exported directly from JMStoolBox: Menu "Help" > "Export JMSToolBox log files..."

**Comment by solace-user** [comment]
> JTB does not process specific metadata outside the JMS specifications and that "message ID" is not explicitely part of the JMS specification JTB can only deal with the information provided via the JMS specs So, if this proprietary info ("message ID") is present in some JMS user property, then you can create a custom "Column Set" to display it in the message browser as any other JMS property Also in that case, you can search messages based on this JMS user property with a selector in the interface
> 
> Columns Sets doc is [here](https://github.com/jmstoolbox/jmstoolbox/wiki/1.6-Columns-Sets)

Sad to say that the JMS Headers or Properties did not indicate that message ID shown in Solace, so custom Column Set cannot help. May I know will you add that feature to JMSToolBox?

**Comment by titou10titou10** [comment]
no. as said before JTB only deal with JMS metadata. If it's not there, then it will not use/display iot

**Comment by pascalre** [comment]
@solace-user the Message ID you see in the PubSub Broker Manager is internal to Solace and cannot be accessed through JMS. You can find more information about this in the [Solace Community](https://solace.community/discussion/43/messageid-versus-jmsmessageid).
